pkg/oomstore/types/types.go

```diff
@@ -1,10 +1,17 @@
 package types
 
+type Category string
+
 const (
-	BatchFeatureCategory  = "batch"
-	StreamFeatureCategory = "stream"
+	BatchCategory  Category = "batch"
+	StreamCategory Category = "stream"
 )
 
+// implement String interface
```

Comment:
The comment doesn't look very necessary since the code is very self-explanatory

Reply:
updated comment